opentelemetry-jaeger/src/exporter/runtime.rs

```diff
@@ -23,8 +26,9 @@ impl JaegerTraceRuntime for opentelemetry::runtime::Tokio {
     type Socket = tokio::net::UdpSocket;
 
     fn create_socket<T: ToSocketAddrs>(&self, endpoint: T) -> thrift::Result<Self::Socket> {
-        let conn = std::net::UdpSocket::bind(bind_addr(&endpoint))?;
-        conn.connect(endpoint)?;
+        let (addrs, family) = addrs_and_family(&endpoint)?;
+        let conn = std::net::UdpSocket::bind(family)?;
+        conn.connect(addrs.as_slice())?;
```

Comment:
Style nit: should be able to just pass `&addrs` instead of calling `as_slice()` explicitly?

Reply:
I did originally attempt to use `&addrs` but rust complained that:
```
the trait 'std::net::ToSocketAddrs' is not implemented for 'std::vec::Vec<std::net::SocketAddr>'
```